sql-to-dbsp-compiler/SQL-compiler/src/test/java/org/dbsp/sqlCompiler/compiler/sql/OtherTests.java

```diff
@@ -595,13 +595,4 @@ public void testRemove() {
         boolean same = change.getSet(0).sameValue(expected);
         Assert.assertTrue(same);
     }
-
-    @Test
-    public void rustFmt() throws IOException, InterruptedException {
```

Comment:
why delete this test?
I use it all the time

Reply:
There's nothing special about this crate that requires these checks for it. Any IDE can be configured to do this, plus you can use a git hook.

Reply:
I can bring it back, but this is just a really weird workflow

Reply:
My Java IDE doesn't do it, and I run these tests when I modify the rust files.
The git hook recommended is really annoying, since it does many other things
I want to only check the files I normally edit.

Reply:
Brought it back, please approve if this looks good otherwise